HousingRepairsOnlineApi.Tests/HelpersTests/AppointmentConfirmationSenderTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Helpers;
+using HousingRepairsOnlineApi.UseCases;
+using Moq;
+using Xunit;
+
+namespace HousingRepairsOnlineApi.Tests.HelpersTests
+{
+    public class AppointmentConfirmationSenderTests
+    {
+        private readonly Mock<ISendAppointmentConfirmationEmailUseCase> sendAppointmentConfirmationEmailUseCaseMock;
+        private readonly Mock<ISendAppointmentConfirmationSmsUseCase> sendAppointmentConfirmationSmsUseCaseMock;
+        private readonly AppointmentConfirmationSender systemUnderTest;
+
+        public AppointmentConfirmationSenderTests()
+        {
+            sendAppointmentConfirmationEmailUseCaseMock = new Mock<ISendAppointmentConfirmationEmailUseCase>();
+            sendAppointmentConfirmationSmsUseCaseMock = new Mock<ISendAppointmentConfirmationSmsUseCase>();
+            systemUnderTest = new AppointmentConfirmationSender(sendAppointmentConfirmationEmailUseCaseMock.Object, sendAppointmentConfirmationSmsUseCaseMock.Object);
+        }
+
+        [Fact]
+        public void GivenEmailContact_WhenExecute_ThenSendAppointmentConfirmationEmailUseCaseIsCalled()
+        {
+            var repairRequest = new RepairRequest { ContactDetails = new RepairContactDetails { Type = "email", Value = "abc@defg.hij" }, Time = new RepairAvailability() { Display = "some time" } };
```

Comment:
Replace "email" with `HousingRepairsOnlineApi.Helpers.Constants.Email` (or it's new name)
```suggestion
            var repairRequest = new RepairRequest { ContactDetails = new RepairContactDetails { Type = HousingRepairsOnlineApi.Helpers.Constants.Email, Value = "abc@defg.hij" }, Time = new RepairAvailability() { Display = "some time" } };
```